pkg/cmd/codespace/create.go

```diff
@@ -380,49 +380,49 @@ func (a *App) handleAdditionalPermissions(ctx context.Context, createParams *api
 		if err := a.browser.Browse(allowPermissionsURL); err != nil {
 			return nil, fmt.Errorf("error opening browser: %w", err)
 		}
-	}
-
-	// Poll until the user has accepted the permissions or timeout
-	err := a.RunWithProgress("Waiting for permissions to be accepted in the browser", func() (err error) {
-		ctx, cancel := context.WithTimeout(ctx, permissionsPollingTimeout)
-		defer cancel()
-
-		done := make(chan error, 1)
-		go func() {
-			for {
-				accepted, err := a.apiClient.GetCodespacesPermissionsCheck(ctx, createParams.RepositoryID, createParams.Branch, location, createParams.DevContainerPath)
-				if err != nil {
-					done <- err
-					return
-				}
 
-				if accepted {
-					done <- nil
-					return
+		// Poll until the user has accepted the permissions or timeout
```

Comment:
Nit: might be nice to pull this block into a function to make it better encapsulated